packages/sanes-chrome-extension/src/routes/account/index.tsx

```diff
@@ -82,6 +83,13 @@ const AccountView = (): JSX.Element => {
         <Block>
           <ListTxs title="Transactions" txs={personaProvider.txs} />
         </Block>
+        <Block marginTop={2}>
+          <Link to="https://support.iov.one/hc/en-us">
```

Comment:
What about storing this link in the cnfig file (especially since we expect changes)?

Reply:
I do not think that we will change it more then once also it will be required to update extension in store in both cases (if we will hardcode URL or even if we will add it to the config file).